redbot/core/utils/chat_formatting.py

```diff
@@ -429,3 +430,29 @@ def humanize_timedelta(
             strings.append(f"{period_value} {unit}")
 
     return ", ".join(strings)
+
+
+def text_to_file(
+    text: str, filename: str = "file.txt", *, spoiler: bool = False, encoding: str = "utf-8"
+):
+    """Prepares text to be sent as a file on Discord, without character limit.
+
+    This writes text into a bytes object that can be used for the ``file`` or ``files`` parameters\
+    of :func:`discord.abc.send`.
+
+    Parameters
+    ----------
+    text: str
+        The text to put in your file.
+    filename: str
+        The name of the file sent. Defaults to ``file.txt``.
+    spoiler: bool
+        Whether the attachment is a spoiler. Defaults to ``False``.
+
+    Returns
+    -------
+    discord.File
```

Comment:
Best to give a description under ``Returns``, even if it is obvious or trivial. I've had cases where Sphinx has whinged because it was missing. If it doesn't do it now, it could well do it in a future version of Sphinx.